### DISCUSSION TITLE
Non-convergence under high ambient pressure

### DISCUSSION THREAD

**Discussion by grabtheNS** [AMReX-Combustion/PeleLMeX#432] [Q&A]
hello,I miss a convergence problem when i try to set ambient pressure to 2atm or more high by

> prob.P_mean = 100000.0 # Ambient pressure in the chamber

it occur 

> [ERROR][rank 0][/home/zhangjiaqian/PelePhysics/Submodules/sundials/src/cvode/cvode.c:3698][cvHandleFailure] At t = 1 and h = 1, the corrector convergence test failed repeatedly or with |h| = hmin.
In my case it's a similar diffusion Bunsen burner setup with the fuel outlet in the center and the oxidizer wrapped around the outside.

I'm guessing it's increased stiff of the chemical source term due to increased pressure, but I don't know the exact cause of the problem.

**Comment by baperry2** [comment]
It is possible that this is due to an increased stiffness issue due to the higher pressure. You can use `amr.fixed_dt` or `amr.init_shrink` in your input file to force a smaller dt and see if that helps.

Depending on what you are doing in the `bcnormal` and `pelelmex_initdata` functions, there could also be an issue if you are specifying states that are inconsistent with the modified pressure. I'd recommend double checking that the density, enthalpy, and compositions that you are specifying in these functions are consistent with your desired pressure.

**Reply by grabtheNS** [reply]
Thank you for your advice! Although I tried reducing the time step size and other measures, I still got an error. The problem may be with the ignition procedure. Fortunately, I found another way to run it normally. This method references Fluent Operating Manual page 2334 for the distribution ignition strategy. First, a steady-state calculation is performed using cold flow, then the same hot air is used for ignition, and finally the desired boundary conditions are set.In my high-pressure test up to 8 atmospheres, they all functioned properly.
